packages/atomic/cypress/e2e/search-box/instant-results/search-box-instant-results.cypress.ts

```diff
@@ -150,10 +150,10 @@ describe('Instant Results Test Suites', () => {
     );
     InstantResultsAssertions.assertHasResultCount(numOfInstantResults);
 
-    CommonAssertions.assertAriaLiveMessageWithoutIt(
+    /*CommonAssertions.assertAriaLiveMessageWithoutIt(
       SearchBoxSelectors.searchBoxAriaLive,
       maxRecentQueriesWithoutQuery.toString()
-    );
+    );*/
```

Comment:
We remove the test or do we enable it back later? (same for others instances)

Reply:
Will re-enable in my next PR, yes. 

Explained in the PR description about the aria-live system.